politeiawww/wsdcrdata.go

```diff
@@ -123,8 +214,97 @@ func newWSDcrdata() (*wsDcrdata, error) {
 	log.Infof("Dcrdata pubsub server version: %v, client version %v",
 		serverSemVer, clientSemVer)
 
+	return c, nil
+}
+
+// reconnect creates a new websocket client, and subscribes to the
+// same subscriptions as the previous client.
+func (w *wsDcrdata) reconnect() error {
+	if w.isShutdown() {
+		log.Errorf("wsDcrdata is shutdown, not reconnecting")
+		return errShutdown
+	}
+
+	done := make(chan struct{})
```

Comment:
Looking at this again, does this channel actually do anything?  Wouldn't this function do the exact same thing if you took the go routine out?

Reply:
Yeah you're right, it's not needed.